starfyre/dom_methods.py

```diff
@@ -91,19 +91,49 @@ def is_attribute(name):
     if not component.is_text_component:
         html += f"{prop_string} >"
 
+   
+    slot_html = ""
+    slot_css = ""
+    slot_js = ""
+  
+
+    is_slot_used = False
     # Render children
     children = component.children
     # childElements.forEach(childElement => render(childElement, dom));
+
+    for element in children:   
+        if element.is_slot_element == True:
+            element.parentElement = component
+            new_html, new_css, new_js = render_helper(element)
+            slot_html += new_html
+            slot_css += new_css
+            slot_js += new_js        
+
     for childElement in children:
-        childElement.parentElement = component
-        new_html, new_css, new_js = render_helper(childElement)
-        html += new_html
-        css += new_css
-        js += new_js
+        if childElement.is_slot_element == False:
+            childElement.parentElement = component            
+            if childElement.tag == "slot":            
+                html += slot_html
+                css += slot_css
+                js += slot_js
+                is_slot_used = True
+            else:
+                new_html, new_css, new_js = render_helper(childElement)
+                html += new_html
+                css += new_css
+                js += new_js 
 
     html += f"</{tag}>\n"
 
     component.html = html
+
+    #check if inner content of component is being used
+    if is_slot_used == False and slot_html != "":
+        html += slot_html
+        css += slot_css
+        js += slot_js
```

Comment:
Can we do three things here
- First, let us add a log line saying that `slot not specified, appending the children to the end of the stack`
- Second, let us somehow name it has adding unspecified children or something similar. Either in a function somewhere or changing the name of the variables.
- let us also change the inline comment saying something like `appending at the end of the stack as slot position not specified`